README.md

```diff
@@ -99,7 +103,8 @@ notebooks](https://github.com/google/jax/tree/main/docs/notebooks).
 
 At its core, JAX is an extensible system for transforming numerical functions.
 Here are four transformations of primary interest: `grad`, `jit`, `vmap`, and
-`pmap`.
+`pmap` (Update: check out a note on `shard_map` - a new more flexible API that supersedes `pmap`
+- above, or simply read the [JEP document](https://jax.readthedocs.io/en/latest/jep/14273-shard-map.html)).
```

Comment:
Same here... and @hawkinsp points out that we should probably emphasize sharded JIT over explicit `shard_map`. So in this context, maybe just removing mentions of `pmap` is the right choice? What do you think?